CODEOWNERS

```diff
@@ -1,3 +1,3 @@
-* @jupiterone/integrations-staff @jupiterone/integrations-contractors
+* @jupiterone/integrations
```

Comment:
![Screenshot 2023-04-24 at 3 57 38 PM](https://user-images.githubusercontent.com/15333061/234102509-6c1132df-2bf0-4479-889c-5571058cee5b.png)

> Unknown owner: make sure the team @jupiterone/integrations exists, is publicly visible, and has write access to the repository

@JupiterOne/security can you please make sure to add `@jupiterone/integrations` as a collaborator at https://github.com/JupiterOne/graph-automox/settings/access